src/include/common/container/lock_free_queue.h

```diff
@@ -0,0 +1,64 @@
+//===----------------------------------------------------------------------===//
```

Comment:
No need to bring in files that are not used - we want everything in terrier to be used and tested. We can always bring these in when we need them.